core-java-modules/core-java-concurrency-simple/src/test/java/com/baeldung/concurrent/synchronize/SychronizedBlockUnitTest.java

```diff
@@ -9,15 +9,15 @@
 
 import static org.junit.Assert.assertEquals;
 
-public class BaeldungSychronizedBlockUnitTest {
+public class SychronizedBlockUnitTest {
```

Comment:
Let's rename this to `SynchronizedBlocksUnitTest` just to be consistent with the original class name

Reply:
Both Classes renamed. Pr updated.